### ISSUE TITLE
Add this package to the official OpenWrt GitHub

### ISSUE THREAD

**Issue by firewire10000** [peter-tank/luci-app-dnscrypt-proxy2#7]
Have you thought about merging your package with the official GitHub repository? There are a lot of people looking for a dnscrypt-proxy2 LuCI app and this would be extremely popular.

**Comment by peter-tank** [comment]
You can try, I maintain this for myself only.

**Comment by firewire10000** [comment]
> You can try, I maintain this for myself only.

I would but someone would have to maintain it and I have no knowledge of creating or maintaining OpenWrt packages.

**Comment by erotavlasme** [comment]
I would be good to have this package since the old [dnscrypt-proxy 2018-11-22-f61ca76a-1](https://github.com/openwrt/packages/tree/openwrt-19.07/net/dnscrypt-proxy) is not maintained.

**Comment by rugk** [comment]
Yeah, I would also like to see this getting official! Want to use the new dnscrypt-proxy! :smiley:
